packages/launchpad/src/setup/OpenBrowser.vue

```diff
@@ -48,7 +42,20 @@ query OpenBrowser {
 }
 `
 
+gql`
+query OpenBrowser_LocalSettings {
+  localSettings {
+    preferences {
+      wasBrowserSetInCLI
+      isValidBrowser
+      globalLaunchCount
+    }
+  }
+}
+`
+
 const query = useQuery({ query: OpenBrowserDocument })
+const lsQuery = useQuery({ query: OpenBrowser_LocalSettingsDocument, requestPolicy: 'network-only' })
```

Comment:
Since the values were being cached by the urql client I couldn't get the information to sync. `globalLaunchCount` was always `0` even thought he value on the server was updating. 

I notice that this may not be the best choice but couldn't really find a better option. I do feel that since most values are cached anyway, having this as a network request when switching views is negligible. But if anyone has a better idea, I'll be more than happy to make the change.